### DISCUSSION TITLE
健链智云 -- 健康数据管理分析预警平台

### DISCUSSION THREAD

**Discussion by fengfengxiong123** [creatorsand/.github#120] [Ideas]
## 一句话 IDEA：

健链智云提供链上健康数据管理，基于历史数据，如相似生活环境、生活习惯、饮食习惯，进行数据分析，为用户提供健康状况预测预警，并提供环境、饮食、运动等指引。

## 我的角色：

项目倡议者

## 其他角色需求（不包含发起人）：

项目v1版本已经构建完毕，现需要app开发（flutter前端）

## 社群昵称：

健链智云

## 个人简介：

6年python开发，擅长解决web前后端的各种项目问题，从零开始构建多个大中小型项目，solana黑客松得过奖。

## 产品是给谁用的？

关注自身健康，尤其是自己或身边亲人有潜在风险的用户。

## 其他产品有什么痛点，用户为什么要选择你的产品？

基于用户的数据，可以不泄密的情况下为用户提供定制服务。

## 当前市场上其他产品存在的痛点是什么，以及你的产品如何独特地解决这些问题，吸引用户选择：

解决医疗行业中 病人的病历数据被篡改，信息丢失不完整等现象
解决医生与患者之间数据交流中效率的问题，使患者共享数据共享更加灵活且具有安全性
使患者够轻松拥有、控制和共享自己的数据且不会被侵犯隐私
 实现数据分布式存储而且不依赖第三方，可以为患者提供全面、不变的日志，增加的身份验证机制使数据访问更加机密性

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
